funnel/models/proposal.py

```diff
@@ -231,6 +232,15 @@ def votes_by_date(self):
                     votes_bydate[groupname][date] += -1 if vote.votedown else +1
         return votes_bydate
 
+    def is_outstation_speaker(self, space_location_geonameid):
```

Comment:
Actually, dump this entire function and make it `location_geonameid` like in `ProposalSpace`. Do the comparison in the template.